parliament/parliament.js

```diff
@@ -202,16 +198,195 @@ function newError (code, msg) {
+   * @property {string} hostname - The hostname of the Parliament instance. Configure the Parliament's hostname to add a link to the Parliament Dashbaord to every alert.
+   */
+
+  /**
+   * The Groups within your Parliament
+   * @typedef ArkimeGroup
+   * @type {object}
+   * @property {string} title - The title of the Group.
+   * @property {string} description - The description of the Group.
+   * @property {Array.<ArkimeClusters>} clusters - The clusters in the Group.
+   */
+
+  /**
+   * The Clusters within your Parliament
+   * @typedef ArkimeCluster
+   * @type {object}
+   * @property {string} title - The title of the Cluster.
+   * @property {string} description - The description of the Cluster.
+   * @property {string} url - The url of the Cluster.
+   * @property {string} localUrl - The local url of the Cluster.
+   * @property {string} type - The type of the Cluster.
+   * @property {string} id - The unique ID of the Cluster.
+   * @property {string} hideDeltaBPS - Whether to hide the delta bits per second of the Cluster.
+   * @property {string} hideDeltaTDPS - Whether to hide the delta packet drops per second of the Cluster.
+   * @property {string} hideMonitoring - Whether to hide number of sessions being recorded of the Cluster.
+   * @property {string} hideMolochNodes - Whether to hide the number of Arkime nodes of the Cluster.
+   * @property {string} hideDataNodes - Whether to hide the number of data nodes of the Cluster.
+   * @property {string} hideTotalNodes - Whether to hide the number of total nodes of the Cluster.
+   */
+
+  /**
+   * GET - /api/parliament
+   *
+   * Retrieves a parliament by id (name).
+   * @name /parliament
+   * @returns {Parliament} parliament - The requested parliament
+   */
+  static async apiGetParliament (req, res) {
+    try {
+      const { body: { _source: parliament } } = await Parliament.getParliament();
+
+      Parliament.#cache.set('parliament', parliament);
+
+      if (!req.user.hasRole('parliamentAdmin')) {
+        delete parliament.settings;
+      }
+
+      return res.json(parliament);
+    } catch (err) {
+      if (ArkimeConfig.debug) {
+        console.log('Error fetching parliament', err);
+      }
+      return res.serverError(500, 'Error fetching parliament');
+    }
+  }
+
+  /**
+   * PUT - /api/parliament
+   *
+   * Updates a parliament by id (name).
+   * @name /parliament
+   * @returns {boolean} success - Whether the operation was successful.
+   * @returns {string} text - The success/error message to (optionally) display to the user.
+   * @returns {Parliament} parliament - The updated parliament.
+   */
+  static async apiUpdateParliament (req, res) {
+    try {
+      await Parliament.setParliament(req.body);
+      return res.json({ success: true, text: 'Parliament updated successfully', parliament: req.body });
```

Comment:
don't just send this back. If I need this on the client, use what I fetched from above